R/derive_extreme_records.R

```diff
@@ -268,6 +282,14 @@ derive_extreme_records <- function(dataset = NULL,
   # Check input arguments
   assert_vars(by_vars, optional = is.null(dataset_ref))
   assert_expr_list(order, optional = TRUE)
+  assert_vars(keep_vars_source, optional = TRUE)
```

Comment:
tidyselect expressions should be accepted.

Reply:
Replaced 'assert_vars()` with `assert_expr_list()` which accepts tidyselect expressions.